cloudformation/ecs-selenium.cfn.yml

```diff
@@ -80,6 +90,10 @@ Parameters:
     Type: String
     Default: 0 0 * * *
     Description: Cron expression for scaling down the nodes to the default capacity (DesiredChromeNodes, DesiredFirefoxNodes).
+  DockerVolumeSize:
+    Type: Number
+    Default: 22
+    Description: Size for EBS volume for Docker image and metadata storage.  # https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-ami-storage-config.html
```

Comment:
It might be helpful to indicate the units (GBs). Either in the description or in the variable name.